### PR TITLE
improve tool configuration via the run command

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

**Review by atheurer — APPROVED** [review]

File: bin/_main [hunk 1]
```diff
@@ -508,20 +512,27 @@ elif [ "${1}" == "run" ]; then
...
+            echo "\"--tool-params=<file>\"."
+            echo ""
+            echo "You can also disable tools entirely using the \"--no-tools\" parameter."
+        else
+            cp ${tool_params} ${tool_params_file}
+        fi
     else
-        cp ${tool_params} ${tool_params_file}
+        echo "Disabling all tools due to --no-tools"
+        echo "[]" > ${tool_params_file}
```
Comment (HughNhan) [inline bin/_main:535]: So .. we will have this behavior ... there is a "custom" tools-params.json in-place, but the user just want to run --no-tools for this one run for whatever reason. Once the --no-tools run runs, the original tools-params.json will be overwritten with "[]".  Is that right?

Comment (k-rister) [reply bin/_main:535]: Yes.  The tool-params.json that is created via the run command (either via copying the default set or by this new method) is always per run.  There is no persistence from run-to-run -- besides whatever files the user themselves has created in their directory/directories.

Comment (atheurer) [reply bin/_main:535]: So your original tool params file is safe/unmodified

**Review by HughNhan — COMMENTED** [review]
LGTM. Thanks

**Pull request merged**